[custom_ops] mutated_args -> mutates_args

zou3519 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #123454
* #123453
* __->__ #123437
* #123129
* #123110
* #123109
* #123108

This seemed better, since when you're construction a custom op you need
to provide "the args that the custom op mutates".

Test Plan:
- existing tests

zou3519 committed:
[custom_ops] mutated_args -> mutates_args

This seemed better, since when you're construction a custom op you need
to provide "the args that the custom op mutates".

Test Plan:
- existing tests

[ghstack-poisoned]

albanD reviewed (approved):
No strong opinion on my end. I'm ok with the change.

zou3519 committed:
Update on "[custom_ops] mutated_args -> mutates_args"


This seemed better, since when you're construction a custom op you need
to provide "the args that the custom op mutates".

Test Plan:
- existing tests

[ghstack-poisoned]

zou3519 committed:
Update on "[custom_ops] mutated_args -> mutates_args"


This seemed better, since when you're construction a custom op you need
to provide "the args that the custom op mutates".

Test Plan:
- existing tests

[ghstack-poisoned]

zou3519 commented:
@pytorchbot merge
